cmd/wallet-sdk-gomobile/walleterror/error.go

```diff
@@ -26,8 +26,8 @@ func Parse(message string) *Error {
 	err := json.Unmarshal([]byte(message), walletErr)
 	if err != nil {
 		return &Error{
-			Code:     "GNR2-000",
-			Category: "GENERAL_ERROR",
+			Code:     "UKN2-000",
+			Category: "OTHER_ERROR",
```

Comment:
Why did we change the category and code here, does it impact the issuance flow error ?

Reply:
Check the commit message for the reason behind this change. This error type only gets returned when there's no underlying error code - we should never be checking for this error code from the app. Note: I do see one place in handle_redirect_uri.dart where the `UKN2-000` is checked, but this appears to be a mistake and I don't think error handling works there as expected. We should fix this when we overhaul the demo app's error handling @rolsonquadras @talwinder50